Gaijinizer patch/Map057.txt

```diff
@@ -252,38 +273,42 @@ However, I was about to be bound all over the body ...! ?#MTLed
 全く存在していないかのような……
 > CONTEXT: Map057/events/5/pages/0/177/Dialogue
 > CONTEXT: Map057/events/5/pages/2/172/Dialogue
-as if………. "Remembering" and "time" between me
-As if it did not exist at all ...#MTLed
+It's as if...
+
+The space between my memory and time doesn't
+exist at all...#RonIffy
 > END STRING
 
 > BEGIN STRING
 逃がしちゃった………
 
 やっぱり、もうちょっと慣れないとダメね……
 > CONTEXT: Map057/events/5/pages/0/183/Dialogue/enemy4
-I missed it ......... After all, I have to get
-used to it a little more ...#MTLed
+I failed...
+
+It seems I must take the time to get more used to
+this...#Ron
 > END STRING
 
 > BEGIN STRING
 ………今度は外さない…
 > CONTEXT: Map057/events/5/pages/0/187/Dialogue/enemy4
-......I will not remove this time ...#MTLed
+I will not let you free this time...#Ron
 > END STRING
```

Comment:
We're missing a word here.... We have two possibilities:
"I will not let you go free this time..."
or
"I will not let you get away this time..."
I prefer the second, but either works.